src/components/converter/Form.js

```diff
@@ -288,6 +291,14 @@ function FormSection() {
       !/[^@]+@[^@]+/.test(email) ||
       !acceptTerms
   )
+
+  const slippageError = useMemo(() => {
+    const isRegistryBalanceZero = balanceAnj.eq(0)
+    const isAmountCloseToSlippageError =
+      amountAnj.gt(SLIPPAGE_LOWER_BOUND) && amountAnj.lt(SLIPPAGE_UPPER_BOUND)
+    return isRegistryBalanceZero && isAmountCloseToSlippageError
```

Comment:
Let's update this logic to be:

- `total amount = balanceAnj + amountAnj`
- `slippageWarning = total amount > minimum anj && (total amount * slippage) < minimum`

Note that a user may have ANJ in the registry, but an amount less than 10k ANJ (if they were slashed). They will now have to activate at least enough ANJ to reach 10k again. And as mentioned above, we don't really care about the upper bound.